libs/map/src/lib/components/action/vl-map-click-action.ts

```diff
@@ -0,0 +1,78 @@
+import {webComponent} from '@domg-wc/common-utilities';
+import {VlMap} from '@domg-wc/map';
+import {LitElement} from "lit";
+import {MapBrowserEvent} from "ol";
+import Overlay from "ol/Overlay";
+import {property} from 'lit/decorators.js';
+import {Coordinate} from "ol/coordinate";
+import {Extent} from "ol/extent";
+import {VlMapClickActionPindrop} from "./vl-map-click-action-pindrop";
+import {Pixel} from "ol/pixel";
+
+/**
+ * VlMapClickAction
+ * @classdesc The map click action component. Adds a marker on the map.
+ * @property {function} onClick - callback die afgevuurd wordt na click op map
+ */
+
+// TODO: crs doorsturen? vb: crs=EPSG:31370 => komt uit map??
+export interface ClickCallback {
```

Comment:
ook een vraag voor UIG Team / Decibel team. OL API (Coordinate, ...) exposen of abstraheren naar eigen types?

Reply:
te bekijken na onderzoek getfeatureinfo call of dit niet onderhevig is aan verandering